app/Http/Client/Alchemy/AlchemyPendingRequest.php

```diff
@@ -421,6 +441,8 @@ public function parseNft(array $nft, int $networkId, bool $convertTokenNumber =
             traits: $this->extractTraits($nft),
             mintedBlock: $nft['contractMetadata']['deployedBlockNumber'],
             mintedAt: $mintTimestamp !== null ? Carbon::createFromTimestampMs($mintTimestamp) : null,
+            hasError: ! empty($error),
+            error: $nftError,
```

Comment:
@patricio0312rev it seems `error` and `hasError` has different meaning while they look like related, at the first glance I thought `error` would be string that shows the actual error while `hasError` is a flag to show the `error` is not `null`, however, it is not the case, could you please give a better names to these?

Reply:
Fair point, maybe we can rename `error` to `info`. So it becomes `NftInfo::MetadataOutdated` and `info: $nftInfo` respectively.

Another option could be `hint`.

(The rename should also apply to the newly added column)